docs/decisions/0016-idv-attempt-events.rst

```diff
@@ -0,0 +1,44 @@
+16. Event definitions for ID Verification
+#########################################
+
+NOTE: This decision doc is still a work in progress.
+
+Status
+******
+
+**Draft** 2024-08-19
+
+Context
+*******
+
+IDV Change:
+===========
+* We're changing it b/c software secure no longer supports IDV
+
+The IDV Vendor:
+===============
+* Persona
+
+
+Decision
+********
+
+Where these events will be produced/consumed:
+=============================================
+
+<!-- This is based off the special exam decision doc, lmk if this is wrong -->
+* `persona-integration` will produce these events.
```

Comment:
Actually, the intention is for the `verify_student` application to create these events, not any individual IDV plugin. You mention that in the "Consequences section" as well.

These events should be vendor agnostic.